tools/build_env/build_env.sh

```diff
@@ -12,23 +13,24 @@ function error() {
 
 function usage() {
   cat <<-EOM
-Usage: $0 [-b] [-p] commands
+Usage: $0 [-b] [-p] [-v variant] commands
   -b (re-)build Docker image
   -p publish Docker image to Docker Hub (you must be logged in)
+  -v target-specific variant, e.g. android
```

Comment:
Good idea! I also renamed the default image `uproxy/build-default` to `uproxy/build-node`.